backend-tests/tests/test_devauth.py

```diff
@@ -74,11 +74,11 @@ def user(clean_migrated_mongo):
 
 @pytest.fixture(scope="function")
 def devices(clean_migrated_mongo, user):
-    uc = ApiClient(useradm.URL_MGMT)
+    useradm_mgmt_v1 = ApiClient(useradm.URL_MGMT)
```

Comment:
I'd split this variable renaming into a dedicated commit.

Reply:
Done!